components/stories/Stars.story.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { action } from '@storybook/addon-actions'
+import { ComponentMeta } from '@storybook/react'
+
+import { Stars } from '../Stars'
+
+export default {
+  component: Stars,
+  title: 'Star Ratings'
+} as ComponentMeta<typeof Stars>
+
+export const Default = () => (
+  <Stars updateCallback={action('Option is selected')} />
```

Comment:
Just flagging here I'm getting a weird return here when the index is 0, & I couldn't figure out why: 
<img width="505" alt="image" src="https://user-images.githubusercontent.com/115499534/232133976-55f1297f-c3e3-45b4-a2a4-4305dcf77ba0.png">

Reply:
Thanks for flagging! I think this is a storybook quirk